source/Weapon.h

```diff
@@ -156,6 +157,7 @@ class Weapon {
 	bool isSafe = false;
 	bool isPhasing = false;
 	bool isDamageScaled = true;
+	bool isParallel = false;
```

Comment:
Let's add a line comment above `isParallel` that indicates what effect this flag has on a given weapon. If someone is reviewing the game's class headers and wondering how they can customize the game's weapons, they shouldn't need to dive all the way into Hardpoint/Armament to see what this flag does.